crates/services/gas_price_service/src/v1/da_source_adapter/service.rs

```diff
@@ -9,19 +9,28 @@ use std::{
     time::Duration,
 };
 use tokio::{
-    sync::mpsc::Sender,
+    sync::broadcast::Sender,
     time::{
         interval,
         Interval,
     },
 };
 
-use crate::v1::da_source_adapter::{
-    DaBlockCosts,
-    POLLING_INTERVAL_MS,
-};
+use crate::v1::da_source_adapter::DaBlockCosts;
 pub use anyhow::Result;
 
+#[derive(Clone)]
+pub struct DaBlockCostsSharedState(Sender<DaBlockCosts>);
```

Comment:
I think "DaBlockCosts" for service and shared state is redundant since it is already live inside of the `v1::da_source_adapter`. 

Also I think "adapter" is overloads the ports and adapters. We don't use this property in context of this file. I think it just can be `da_source` or `da_soruce_service`.

Reply:
not sure I understand, are you asking me to rename something?

Reply:
okay I re-read it ~ renaming it

Reply:
addressed in 0ad219125f